.github/workflows/ci.yml

```diff
@@ -1135,16 +1135,24 @@ jobs:
                     docker/sdk up -t
                     docker/sdk cli composer dump-autoload -o -a --apcu
                     docker/sdk cli console queue:worker:start --stop-when-empty
-            -   name: Run Tests
+            -   name: Init test env
                 run: |
                     docker/sdk exec cypress-tests cp .env.example .env
+            -   name: Run Tests
+                run: |
                     docker/sdk exec --env "ENV_REPOSITORY_ID=b2b" --env "ENV_IS_SSP_ENABLED=true" cypress-tests npm run cy:ci
-                    docker/sdk exec --env "ENV_REPOSITORY_ID=b2b" cypress-tests npm run cy:ci:ssp
             -   name: Upload artifacts
                 if: failure()
                 run: |
                     AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .cypress s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2b/dms-off/cypress/${GITHUB_RUN_ID}/PHP8.3MariaDB/ --recursive
-
+            -   name: Run SSP Tests
+                run: |
+                    docker/sdk exec --env "ENV_REPOSITORY_ID=b2b" cypress-tests npm run cy:ci:ssp
+                if: always()
+            -   name: Upload SSP artifacts
+                if: failure()
+                run: |
```

Comment:
please be sure this part will not override previous step artifacts (core ssp)

Reply:
These changes insure that all artifacts are stored )

Reply:
<img width="496" height="249" alt="Screenshot 2025-07-17 at 21 52 15" src="https://github.com/user-attachments/assets/cc2a8e65-c40e-40c5-b95b-4d7fe4abdcfb" />